database.rules.json

```diff
@@ -1,6 +1,9 @@
 {
   "rules": {
     ".read": true,
-    ".write": true
+    ".write": true,
+    "books": {
+      ".indexOn": ["library","author"]
+    }
```

Comment:
What does this do?

Reply:
Firebase generated a few warnings in console and suggested to add `.indexOn` to Firebase config. Probably the database engine adds indexes to documents so the lookup for referenced record can be faster.

Reply:
More information about `.indexOn` on Firebase: https://firebase.google.com/docs/database/security/indexing-data